app/components/general/components/bc-button/macro.test.js

```diff
@@ -33,6 +33,19 @@ describe('bcButton', () => {
     });
   }));
 
+  it('should render with the button text provided as an aria-label if altText is not provided', componentTester(setup, (harness) => {
```

Comment:
I think the button component does this when it is not disabled anyway?

Reply:
nah it doesn't

Reply:
this sets it on the outer div rather than the button component itself.